styles.css

```diff
@@ -0,0 +1,23 @@
+@media (max-width: 80em) {
+    
+    html {
+      font-size: 100%;
+    }
+  }
+
+  
+  #welcome-section {
+      display: inline-flex;
+      background-color: cornflowerblue;
+      text-align: center;
+  }
+  #navbar {
+    position: fixed;
+    width: 100%;
+    top: 0;
+    left: 0;
+    color: white;
+    font-family: 'Exo 2', sans-serif;
+    padding: 10px;
+    margin-left: 40px;
+  }
```

Comment:
The product landing page should utilize CSS flexbox at least once and should have at least one media query as required in the project. Please utilize CSS flexbox and media query.